docker-compose.yaml

```diff
@@ -3,6 +3,32 @@ version: '3.4'
 services:
     db:
         image: postgres:11.2-alpine
+    s3:
+        image: minio/minio:RELEASE.2019-05-02T19-07-09Z
+        environment:
+            MINIO_ACCESS_KEY: &AWSKey AKIAIOSFODNN7EXAMPLE
+            MINIO_SECRET_KEY: &AWSSecret wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+            MINIO_REGION: &AWSRegion us-east-1
+        command: server /app
```

Comment:
assuming this is storing data locally as a stand-in for S3, should it mount a (anonymous) volume to show that?

Reply:
Since it's for local development I hadn't. (We don't do it for the database too.)

Something that could be done later if proved useful.